OrdersApplication/OrdersSectionCell/OrdersSectionCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  OrdersSectionCell.swift
+//  OrdersApplication
+//
+//  Created by Esraa Khaled   on 23/09/2022.
+//
+
+import UIKit
+
+class OrdersSectionCell: UITableViewCell {
+
+    @IBOutlet weak var orderImage: UIImageView!
```

Comment:
```suggestion
    @IBOutlet private(set) weak var orderImage: UIImageView!
```
This applies to the rest properties.